src/backend/marsha/settings.py

```diff
@@ -405,6 +405,7 @@ class Base(Configuration):
             },
         }
     )
+    REMINDERS_SECRET_HASH = values.Value("Marsha")
```

Comment:
```suggestion
    REMINDERS_UNSUBSCRIBE_SECRET = values.SecretValue()
```
A secret value as no default and you should define it in the environment variable (`env.d/development.dist` and `env.d/test`)

Reply:
For the name, we will use this hash for other purposes such as getting access to the video, I wouldn't use unsubscribe in the name not to make it specific